screens/StartScreen.tsx

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
```

Comment:
Most of these imports are unused -- make sure you have your linter configurations on so that unused imports are removed!